src/app/test-record/test-history/test-history.component.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-summary-list {
```

Comment:
instead of writing all these custom CSS use this option govuk-!-margin-bottom-3. Investigate and see the one that will work and will remove or reduce the margin. Once found please remove all the custom CSS

Reply:
I did try, but gov uk has only for margin and padding, not for border as it is needed to be compliant with our design